.travis.yml

```diff
@@ -16,7 +16,7 @@
 
 language: sh
 os: linux
-dist: xenial
+dist: focal
```

Comment:
PR https://github.com/ARMmbed/mbed-os/pull/14538 adds CMake based on the Ubuntu version, so this would break it. Just a heads up we'll need to coordinate merging those.